app/src/main/java/io/apicurio/registry/rest/v2/GroupsResourceImpl.java

```diff
@@ -597,15 +599,33 @@ public ArtifactMetaData createArtifact(String groupId, ArtifactType xRegistryArt
     }
 
     /**
-     * @see io.apicurio.registry.rest.v2.GroupsResource#createArtifactWithRefs(String, ArtifactType, String, String, IfExists, Boolean, String, String, String, String, ContentCreateRequest)
+     * @see io.apicurio.registry.rest.v2.GroupsResource#createArtifact(java.lang.String, io.apicurio.registry.types.ArtifactType, java.lang.String, java.lang.String, io.apicurio.registry.rest.v2.beans.IfExists, java.lang.Boolean, java.lang.String, java.lang.String, java.lang.String, java.lang.String, io.apicurio.registry.rest.v2.beans.ContentCreateRequest)
```

Comment:
While a small thing, absolute imports of primitive types seems like unnecessary noise.

Reply:
Yeah - that's just what Eclipse does when generating that comment.  🤷